src/thinblock.cpp

```diff
@@ -28,6 +28,26 @@ extern CCriticalSection cs_thinblockstats;
 extern CCriticalSection cs_orphancache;
 extern map<uint256, COrphanTx> mapOrphanTransactions GUARDED_BY(cs_orphancache);
 
+string formatInfoUnit(double value)
+{
+    static const char *units[] = {"B", "KB", "MB", "GB", "TB", "PB", "EB"};
+
+    if (value < 0)
+        value = 0.0;
```

Comment:
Ok so you want -10000 be displayed at -10.00KB for example?

Sure I can do that. Now I just wonder: does it make sense to move the whole function somewhere else, like generic util.cpp, or unlimited.cpp?

Or is it fine where it is?